testfiles/gedcom70/sour-1.ged

```diff
@@ -0,0 +1,47 @@
+﻿0 HEAD
+1 GEDC
+2 VERS 7.0
+0 @X1@ SOUR
+1 NOTE A source described inline
+0 @X2@ SOUR
+1 NOTE Another source described inline
```

Comment:
Confusing text that does not describe the file (more like it below too)

Reply:
Updated